JS/Lesson35/onur/my-app/src/App.jsx

```diff
@@ -1,41 +1,61 @@
-import './App.css';
-import { useEffect, useState } from 'react';
-import { BD_SEARCH_BASE_URL } from './urls.js';
-import { AppRouter } from './AppRouter.jsx';
-import { Navbar } from './components/Navbar'
-
-export const App = () => {
-	// 1. Fetch recipes data from an API (Get request, API key, useState to store data, useEffect)
-	// 2. Display the data (RecipeList component -> use <ul> to display data)
-	// 3. Search recipe (A new component SearchBar with a <form> element, text input, and submit functionality)
+import { Box, ThemeProvider, createTheme } from "@mui/material";
+import { Navbar } from "./components/Navbar";
+import { useEffect, useState } from "react";
+import { BD_SEARCH_BASE_URL } from "./urls.js";
+import { AppRouter } from "./AppRouter";
 
+const App = () => {
 	const [recipes, setRecipes] = useState([]);
-	// If you want to keep both initial recipes and recipes from search  on the screen at the same time use code below:
-	// const [searchResult, setSearchResult] = useState([]);
+	const [originalRecipes, setOriginalRecipes] = useState([]);
 
-	// Runs once of component render (component render = "sayfanin yenilenmesi")
-	useEffect(() => {
-		fetch(BD_SEARCH_BASE_URL)
-			.then((response) => response.json())
-			.then((data) => setRecipes(data.meals));
-	}, []);
+	const [mode, setMode] = useState("light");
```

Comment:
Going forward, since you know Typescript now, you should use enum type here instead with values Mode.Light and Mode.Dark. No need to change anything here for this assignment. :)